csrc/src/filtered_lrelu.cpp

```diff
@@ -11,15 +11,17 @@
 //#include <torch/extension.h>
 #include <ATen/cuda/CUDAContext.h>
 #include <c10/cuda/CUDAGuard.h>
-#include "lantern_ob.h"
 #include "styleganr/styleganr.h"
 #include "filtered_lrelu.h"
 #include <torch/torch.h>
+#define LANTERN_TYPES_IMPL // should only be defined in a single file
```

Comment:
I think remove this line because it is already defined in `types.cpp` (I had to remove it to get this to compile locally)